doc/spec/interpretability.rst

```diff
@@ -0,0 +1,95 @@
+Interpretability
+================
+
+Our package offers multiple interpretability tools to better understand the final model CATE.
+
+
+Tree Interpreter
+----------------
+
+Tree Interpreter provides a presentation-ready summary of the key features that explain the biggest differences in responsiveness to an intervention.
+
+:class:`.SingleTreeCateInterpreter` trains a single shallow decision tree for the treatment effect :math:`\theta(X)` you learned from any of
+our available CATE estimators on a small set of feature :math:`X` that you are interested to learn heterogeneity from. The model will split on the cutoff
+points that maximize the treatment effect difference in each leaf. Finally each leaf will be a subgroup of samples that respond to a treatment differently
+from other leaves. 
+
+For instance: 
+
+.. testsetup::
+
+    import numpy as np
+    X = np.random.choice(np.arange(5), size=(100,3))
+    Y = np.random.normal(size=(100,2))
+    y = np.random.normal(size=(100,))
+    T = np.random.choice(np.arange(3), size=(100,2))
+    t = T[:,0]
+    W = np.random.normal(size=(100,2))
+    
+
+.. testcode::
+
+    from econml.cate_interpreter import SingleTreeCateInterpreter
+    from econml.dml import LinearDML
+    est = LinearDML()
+    est.fit(y, t, X=X, W=W)
+    intrp = SingleTreeCateInterpreter(include_model_uncertainty=True, max_depth=2, min_samples_leaf=10)
+    # We interpret the CATE model's behavior based on the features used for heterogeneity
+    intrp.interpret(est, X)
+    # Plot the tree
+    intrp.plot(feature_names=['A', 'B', 'C'], fontsize=12)
+
+Policy Interpreter
+------------------
+Policy Interpreter offers similar functionality but taking cost into consideration. 
```

Comment:
Maybe change the wording here. The policy interpreter, instead of fitting a tree to learn groups that have a different treatment effect, it tries to split the samples into different treatment groups. So in the case of binary treatments it tries to create sub-groups such that all samples within the group have either all positive effect or all negative effect. Thus it tries to separate responders from non-responders, as opposed to trying to find groups that have different levels of response. 

This way you can construct an interpretable personalized policy where you treat the groups with a postive effect and don't treat the group with a negative effect. Our policy tree provides the recommended treatment at each leaf node.

Reply:
Done!